release-notes/7.0/preview/api-diff/preview2/Microsoft.NETCore.App/7.0-preview2_System.Data.SqlTypes.md

```diff
@@ -0,0 +1,71 @@
+# System.Data.SqlTypes
+
+``` diff
+ namespace System.Data.SqlTypes {
+-    public struct SqlBinary : IComparable, INullable, IXmlSerializable {
++    public struct SqlBinary : IComparable, IEquatable<SqlBinary>, INullable, IXmlSerializable {
++        public bool Equals(SqlBinary other);
+-        XmlSchema IXmlSerializable.GetSchema();
++        XmlSchema System.Xml.Serialization.IXmlSerializable.GetSchema();
+-        void IXmlSerializable.ReadXml(XmlReader reader);
++        void System.Xml.Serialization.IXmlSerializable.ReadXml(XmlReader reader);
+-        void IXmlSerializable.WriteXml(XmlWriter writer);
++        void System.Xml.Serialization.IXmlSerializable.WriteXml(XmlWriter writer);
```

Comment:
These IXmlSerializable diffs are just different renderings of the same metadata.  We had this same problem in a bunch of places with the Preview 1 diffs.  We should revert all such diffs here as they're not actually representative of a real difference. (And if we don't already know the cause, we should figure it out and fix the issue for next time :smile:)

(The IEquatable changes in this file are real, however, and should remain.)